This patch makes it possible to install the implementation files to
the site-packages directory of choice, under a directory "qarte"

=== modified file '__init__.py'
--- old/__init__.py	2016-03-21 04:11:28 +0000
+++ new/__init__.py	2017-02-21 12:48:59 +0000
@@ -0,0 +1,2 @@
+import os,sys
+os.path.dirname(os.path.abspath(__file__))

=== modified file 'qarte'
--- old/qarte	2017-02-08 09:43:25 +0000
+++ new/qarte	2017-02-21 12:56:34 +0000
@@ -31,6 +31,8 @@
 import platform
 import locale
 
+import qarte
+
 VERSION_STR = "3.6.1"
 
 from PyQt5.QtWidgets import QApplication
@@ -81,15 +83,16 @@
             show_infos(args[1])
 
         if args[1] == "-a":
-            os.chdir("/usr/share/qarte")
-
-            if not os.path.isfile("/usr/share/qarte/daemon.py"):
-                logger.critical("File '/usr/share/qarte/daemon.py' not found.")
-                logger.critical("Please re-install Qarte")
-                sys.exit()
-
-            sys.path.append("/usr/share/qarte")
-            from daemon import Daemon
+            #os.chdir("/usr/share/qarte")
+
+            #if not os.path.isfile("/usr/share/qarte/daemon.py"):
+                #logger.critical("File '/usr/share/qarte/daemon.py' not found.")
+                #logger.critical("Please re-install Qarte")
+                #sys.exit()
+
+            #sys.path.append("/usr/share/qarte")
+            os.chdir(qarte.__path__[0])
+            from qarte.daemon import Daemon
             qarte = Daemon(args[2])
             reply = qarte.read_task()
             sys.exit()
@@ -97,20 +100,21 @@
         if args[1] in ["-d", "--debug"]:
             print_infos()
 
-    os.chdir("/usr/share")
-    if not os.path.isdir("qarte"):
-        logger.critical("Folder '/usr/share/qarte' not found.")
-        logger.critical("Please install Qarte in '/usr/share/'")
-        sys.exit()
-
-    os.chdir("/usr/share/qarte")
-    if not os.path.isfile("core.py"):
-        logger.critical("File '/usr/share/qarte/core.py' not found.")
-        logger.critical("Please re-install Qarte")
-        sys.exit()
-
-    sys.path.append("/usr/share/qarte")
+    #os.chdir("/usr/share")
+    #if not os.path.isdir("qarte"):
+        #logger.critical("Folder '/usr/share/qarte' not found.")
+        #logger.critical("Please install Qarte in '/usr/share/'")
+        #sys.exit()
+
+    #os.chdir("/usr/share/qarte")
+    #if not os.path.isfile("core.py"):
+        #logger.critical("File '/usr/share/qarte/core.py' not found.")
+        #logger.critical("Please re-install Qarte")
+        #sys.exit()
+
+    #sys.path.append("/usr/share/qarte")
+    os.chdir(qarte.__path__[0])
     app = QApplication(args)
-    from core import Core
+    from qarte.core import Core
     main = Core()
     sys.exit(app.exec_())


